edx-platform/pearson-vue-theme/lms/templates/student_account/login.underscore

```diff
@@ -1,70 +1,62 @@
 <div class="background-login"></div>
 
-<div>
+<div class="login-container">
   <img class="signin-logo" src="/static/pearson-vue-theme/images/CertPREP_Logo.png">
-</div>
-
-<div class="signin-container">
-  <div class="js-form-feedback" aria-live="assertive" tabindex="-1"></div>
 
 
-  <div class="toggle-form">
-    <span class="text"><%- gettext("First time here?") %></span>
-    <a href="#login" class="form-toggle" data-type="register"><%- gettext("Create an account.") %></a>
-  </div>
+  <div class="signin-container">
+    <div class="js-form-feedback" aria-live="assertive" tabindex="-1"></div>
 
 
-  <h2><%- gettext("Sign in") %></h2>
+    <div class="toggle-form">
+      <h1><%- gettext("Welcome!") %></h1>
+      <br>
+      <span class="text"><%- gettext("Sign in or create an account to access your courseware") %></span>
```

Comment:
Is this the required text? @luismorenolopera Please add a period at the end of the line if so.

Reply:
I have left an image as a reference for a 480px screen @Squirrel18

Reply:
yes, is required

Reply:
So plaet add the period at the end of the line.

Reply:
done